### PR TITLE
DOC: Active Contour, snake length depends on input

### PR DESCRIPTION
**Pull request by anshitag**
## Description
Updated documentation of active_contour that snake length depends on input.

## References
Documentation, it closes issue #2093

### CODE REVIEW

File: skimage/segmentation/active_contour_model.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ def active_contour(image, snake, alpha=0.01, beta=0.1,
     Active contours by fitting snakes to features of images. Supports single
     and multichannel 2D images. Snakes can be periodic (for segmentation) or
     have fixed and/or free ends.
+    The output snake length is the same as the input boundary given--no points
```
Comment (soupault) [inline skimage/segmentation/active_contour_model.py]: Is `--` being rendered somehow specially?

Comment (anshitag) [inline skimage/segmentation/active_contour_model.py]: No, it can be removed

Comment (soupault) [inline skimage/segmentation/active_contour_model.py]: @anshitag Why? Sorry, I don't understand.

Comment (anshitag) [inline skimage/segmentation/active_contour_model.py]: It's just text, `--` is texturally equivalent to `thus`. It can be changed.

Comment (soupault) [inline skimage/segmentation/active_contour_model.py]: I'd be in favour of pure text version :) :
```
The output snake has the same length as the input boundary.
As the number of points is constant, make sure that the initial
snake has enough points to capture the details of the final contour.
```

Comment (anshitag) [inline skimage/segmentation/active_contour_model.py]: Thanks, will make the required changes :)

**Pull request merged**